tools/simulator/frameworks/runtime-src/Classes/ide-support/RuntimeJsImpl.cpp

```diff
@@ -29,6 +29,10 @@
 //
 //
 
+#ifdef WIN32
+#define _HAS_STD_BYTE 0
+#endif
```

Comment:
It is better to add some comment to explain it.

Reply:
added